tests/unit/test_api.py

```diff
@@ -38,8 +38,8 @@ def build_dict(owner, copr_url):
                 "fedora-rawhide-x86_64",
             ],
             "ended_on": 1566377991,
-            "id": 1010428,
-            "ownername": f"{owner}",
+            "id": f"{id}",
```

Comment:
Is there some reason for f-strings?

```suggestion
            "id": str(id)",
```

Reply:
The only reason is that f-strings cool :sunglasses: